docs/api-reference-source/npm.xml

```diff
@@ -70,6 +70,20 @@ $ lemon --pin-packages script.js
 			]]>
 			</discussion>
 		</detail>
+		<detail name="Requiring other files" id="specifying-versions">
+			<discussion>
+			<![CDATA[
+				<p>
+					If your script needs a use a specific sub-file of a package, rather than the package's default, you can specify it using a colon to separate the package name from the path.
```

Comment:
Let's add a bit more context still.

```suggestion
					If your script needs to use a specific sub-file of a package, rather than the package's main file, you can specify it by using a colon to separate the package name from the sub-file path.
```